rcl_interfaces/msg/ParameterValue.msg

```diff
@@ -6,8 +6,8 @@
 uint8 type
 
 # "Variant" style storage of the parameter value.
-bool bool_value
-int64 integer_value
-float64 double_value
-string string_value
-byte[] bytes_value
+bool bool_value False
+int64 integer_value 0
+float64 double_value 0.0
+string string_value ""
+byte[] bytes_value []
```

Comment:
The default value for the array is unnecessary. An array is always empty by default.